### PR TITLE
Add variables for header height

### PR DESCRIPTION
**Pull request by mdubus**
# Pull Request

## What does this PR do?
With the addition of the banner on top of the header, some behaviours were broken. For example, on the legal pages, we have a sticky menu on the left which position is determined by the header height ; and which goes below the header on scroll because the header is now bigger (banner added).
Problem on video: 

https://user-images.githubusercontent.com/30866152/197784523-434afe82-3b04-4bd3-ac4c-1eabde303409.mov

In this PR, we have now a variable for the header height (both in mobile and desktop) + for the banner height, which is more maintainable than hard-coded values.

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: src/layouts/LegalLayout.js [hunk 1]
```diff
@@ -36,18 +40,22 @@ const GridWrapper = styled.div`
 const StickyWrapper = styled.div`
   @media (min-width: ${get('breakpoints.md')}) {
     position: sticky;
-    top: calc(98px + 54px);
-  }
-  @media (min-width: ${get('breakpoints.lg')}) {
-    top: calc(88px + 54px);
+    top: calc(
+      var(--header-height-desktop) + 54px
```
Comment (bidoubiwa) [inline src/layouts/LegalLayout.js:44]: what represents the `54px` ?

Comment (mdubus) [reply src/layouts/LegalLayout.js:44]: It's just an extraneous space in order to not have the sticky menu sticking to the header, but instead being a little bit lower :)

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Comment by mdubus** [comment]
bors merge

**Pull request merged**